entries/c/changelly.com.json

```
@@ -2,9 +2,11 @@
   "Changelly": {
     "domain": "changelly.com",
     "tfa": [
-      "totp"
+      "totp",
+      "u2f"
```

```suggestion
      "custom-hardware"
    ],
    "custom-hardware": [
      "Yubico OTP"
```

The documentation [lists support for Yubico OTP](https://support.changelly.com/en/support/solutions/articles/14000111543), but not FIDO2/U2F.

> Changelly PRO supports all YubiKey devices pre-configured with Yubico-OTP function